src/DataProtection/docs/GetBackupFrequenceFromTimeInterval.md

```diff
@@ -0,0 +1,73 @@
+---
+external help file:
+Module Name: Az.DataProtection
+online version: https://docs.microsoft.com/powershell/module/az.dataprotection/getbackupfrequencefromtimeinterval
+schema: 2.0.0
+---
+
+# GetBackupFrequenceFromTimeInterval
+
+## SYNOPSIS
+
+
+## SYNTAX
+
+```
+GetBackupFrequenceFromTimeInterval [-RepeatingTimeInterval] <String[]> [<CommonParameters>]
+```
+
+## DESCRIPTION
+
+
+## EXAMPLES
+
+### Example 1: {{ Add title here }}
+```powershell
```

Comment:
Please populate all examples like these.

Reply:
Hi @dolauli  these are helper scripts which will not be exposed to customers. We use these inside our customized cmdlets and these are not part of Az.DataProtection.psd1. With that in mind, do we still need to give examples for these?

Reply:
This is for more information. We've distinguised these helper scripts from actual cmdlets by not  having "-" as a separator between Verb and Subject

Reply:
Add `[Microsoft.Azure.PowerShell.Cmdlets.DataProtection.DoNotExportAttribute()]` as below for all the help functions
```
function AddFilterToQuery {
	[Microsoft.Azure.PowerShell.Cmdlets.DataProtection.DoNotExportAttribute()]
```